config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -245,7 +229,7 @@ presubmits:
           - e2e_test_gke_v2.yaml
           - --test_result_folder
           - e2e_test
-    - name: presubmit-sample-test-v2
+    - name: presubmit-sample-test
```

Comment:
Can you follow the naming convention in this file "<repo>-<test descriptor>".
This makes it easier to identify tests in prow jobs dashboard and other places.
https://prow.k8s.io/?repo=kubeflow%2Fkubeflow

Reply:
done